frontend-mobile/lib/widgets/cubit/coffee_call_cubit.dart

```diff
@@ -53,15 +54,15 @@ class CoffeeCallCubit extends Cubit<CoffeeCallState> {
     socket.init();
   }
 
-  void join() {
-    protocol.join('Zaphod Beeblebrox');
```

Comment:
Again, this would be a nightmare to test. If this were a serious project I'd suggest to inject a wrapper or _at least_ the wrapper itself into the cubit instead of dependening on it directly.

Reply:
Thanks for the hint. I will introduce an own cubit for the settings in a new pr